models/valve_landmarks/configs/metadata.json

```
@@ -12,6 +13,7 @@
     "pytorch_version": "1.13.0",
     "numpy_version": "1.21.2",
     "optional_packages_version": {},
+    "name": "Valve landmarks regression",
```

```suggestion
    "name": "Valve Landmarks Regression",
```
I would suggest here and elsewhere that names should be in title case as this would look best on the model zoo website and elsewhere.

---

Hi @ericspod @yiheng-wang-nv , I am OK with both naming formats, but please try to unify all the bundle names.

Thanks.

---

Thanks @ericspod , as @Nic-Ma suggested, I'm not sure if we should capitalize all words, thus I followed the rule of our current webpage and only keep the first word capital. What do you think?

---

Hi @ericspod , how about merging this PR, and if needing to use capital case on all words, feel free to submit a new ticket to discuss? Thanks!

---

@yiheng-wang-nv  Yes let's look at that later and put this through for now.